pandas/tests/groupby/test_categorical.py

```diff
@@ -1252,3 +1252,29 @@ def test_get_nonexistent_category():
                 {"var": [rows.iloc[-1]["var"]], "val": [rows.iloc[-1]["vau"]]}
             )
         )
+
+
+@pytest.mark.parametrize("aggregation", ["sum", "mean", "min", "count"])
```

Comment:
Instead of doing this can you use the `reduction_func` fixture that already exists in the groupby conftest?

Reply:
Cool, didn't know about that one! This exposed some other reductions that don't respect the`observed` keyword, so I'll try to correct those as well.